core/src/main/java/com/backblaze/b2/json/B2Json.java

```diff
@@ -139,23 +140,35 @@ public static B2Json get() {
         }
     }
 
+    public void toJson(Object obj, OutputStream out) throws IOException, B2JsonException {
+        toJson(obj, B2JsonOptions.DEFAULT, out);
+    }
+
+    public void toJson(Object obj, B2JsonOptions options, OutputStream out) throws IOException, B2JsonException {
+        toJson(obj, options, out, null);
+    }
+
     /**
      * Turn an object into JSON, writing the results to given
      * output stream.
      *
+     * objTypeOrNull can be set to null if obj is not a parameterized class. However,
+     * if obj contains type parameters (like if obj is a {@literal List<String>}, then
+     * you will need to pass in its type information via objTypeOrNull. This will instruct
+     * B2Json to derive the B2JsonTypeHandler from the type information instead of the
+     * object's class.
```

Comment:
Maybe give an example of how to get the type handler if I have a `List<String>`?

Reply:
Done